### ISSUE TITLE
find missing word

### ISSUE THREAD

**Issue by Fredouh** [PyWaves/pywavestools#2]
Hi, I have a seed of 14 words so one word is missing. I dont know the position of that word. 
Can the script be changed so I type in my 14 words I have and it runs through the 30.720 options of 2048 words in 15 possible positions? I also have my public key.

**Comment by sabotagebeats** [comment]
yes i think it would be, you would need to do an iterator to find it until the generated public key is equal to the current public key.
